tests/testDetectAndMeasure.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+from __future__ import absolute_import, division, print_function
+#
+# LSST Data Management System
+# Copyright 2008-2013 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+"""Test DetectAndMeasureTask
+"""
+import os
+import unittest
+
+import lsst.utils.tests
+from lsst.utils import getPackageDir
+from lsst.afw.geom import Box2I, Point2I, Extent2I
+from lsst.afw.table import SourceTable
+from lsst.daf.persistence import Butler
+from lsst.meas.algorithms.testUtils import plantSources
+from lsst.pipe.tasks.detectAndMeasure import DetectAndMeasureTask
+
+try:
+    obsTestDir = getPackageDir("obs_test")
+    InputDir = os.path.join(obsTestDir, "data", "input")
+except Exception:
+    InputDir = None
+
+class ProcessCcdTestCase(lsst.utils.tests.TestCase):
```

Comment:
Why is this called `ProcessCcdTestCase` when it's supposed to be testing `DetectAndMeasureTask`?